### ISSUE TITLE
While porting between levels commmands editor goes unresponsive; "pops-under" main window

### ISSUE THREAD

**Issue by ghramsey** [MattFiler/OpenCAGE#312]
Porting takes quite a while and the window goes to "not responding" whenever it's clicked on.    This might be expected behavior.

Maybe some sort of status indicator would be better though  The window shows what is being ported, but is too small to display everything so it's being underutilized in it's current form.   I would either do away with it or make it large enough to show it's contents.

The status window for the porting process is not locked into focus.
It "pops-under" the main window making the whole program unresponsive for a time including inablity to use minimize.

![image](https://github.com/MattFiler/OpenCAGE/assets/51859157/52c14e6d-8c8a-4896-9be6-4a335954d182)

Even once the process if finished. if the status window is beneath the main window it does not re-focus to be on top leaving
no indicator that it's finished.  I had to use alt-tab to locate it if I tab away from it while it's working.

I know porting is still jank and being worked on.  Hopefully in future versions these issues can be worked on.

**Comment by MattFiler** [comment]
Yeah I plan on there being big optimisations to the porting functionality before I consider it out of the experimental stage - but thanks for flagging!

**Comment by ghramsey** [comment]
As an addition to this the porting window when finished does not come into focus.  It remains popped under the main window.
I thought the program was hung until I looked in task manager to see it wasn't.  Task Manager let me bring the porting window to focus so I could click OK.

**Comment by MattFiler** [comment]
Gonna close this as its closely related to #247
